terraform/aws-accounts/cloud-platform-aws/vpc/eks/components/components.tf

```diff
@@ -105,7 +106,7 @@ module "modsec_ingress_controllers_v1" {
   replica_count       = "6"
   controller_name     = "modsec"
   cluster_domain_name = data.terraform_remote_state.cluster.outputs.cluster_domain_name
-  is_live_cluster     = lookup(local.prod_workspace, terraform.workspace, false)
+  is_live_cluster     = terraform.workspace == "live" ? true : false
```

Comment:
Should `live-2` also be true for ``is_live_cluster?`

Reply:
This flag is used to determine external-dns-annotation to serve hosts with `*.cloud-platform.service.justice.gov.uk`. I dont think we share the same hosts under *.cloud-platform. between 2 clusters. So I have left the live-2 here.